docs/pages/guard.page.server.mdx

```diff
@@ -7,5 +7,5 @@ The most common use cases being authentication and authorization, see <Link href
 Note that:
 
 - It's always used together with <Link href="/render" text={<code>throw render()</code>} /> or <Link href="/redirect" text={<code>throw redirect()</code>} />. (The `guard()` hook doesn't accept any return value.)
-- It can be asynchronous. (Unlike Route Functions which are always synchronous.)
```

Comment:
FYI this seems not to be the case anymore, since we have now `RouteSync` and `RouteAsync`, so I removed this sentence.

Reply:
Async Route Functions are deprecated and will be removed in the next major release.

Instead of removing the whole sentence, maybe we can remove `always` to make it less misleading.

```diff
- (Unlike Route Functions which are always synchronous.)
+ (Unlike Route Functions which are synchronous.)
```

Reply:
:+1: fixed